biorxiv/corpora_comparison/corpora_comparison.py

```diff
@@ -0,0 +1,668 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Comparative Linguistic Analysis of bioRxiv and PMC
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+from collections import defaultdict, Counter
+import csv
+import itertools
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pickle
+import spacy
+from scipy.stats import chi2_contingency
+from tqdm import tqdm_notebook
+
+from annorxiver_modules.corpora_comparison_helper import(
+    aggregate_word_counts,
+    dump_to_dataframe,
+    get_term_statistics,
+    KL_divergence
+)
+
+
+# # Full Text Comparison (Global)
+
+# ## Gather Word Frequencies
+
+# In[2]:
+
+
+biorxiv_count_path = Path("output/total_word_counts/biorxiv_total_count.tsv")
+pmc_count_path = Path("output/total_word_counts/pmc_total_count.tsv")
+nytac_count_path = Path("output/total_word_counts/nytac_total_count.tsv")
+
+
+# In[3]:
+
+
+if not biorxiv_count_path.exists():
+    biorxiv_corpus_count = (
+        aggregate_word_counts(
+            list(Path("output/biorxiv_word_counts").rglob("*tsv"))
+        )
+    )
+    dump_to_dataframe(biorxiv_corpus_count, "output/biorxiv_total_count.tsv")
+    biorxiv_corpus_count.most_common(10)
+
+
+# In[4]:
+
+
+if not pmc_count_path.exists():
+    pmc_corpus_count = (
+        aggregate_word_counts(
+            list(Path("../../pmc/pmc_corpus/pmc_word_counts").rglob("*tsv"))
+        )
+    )
+    dump_to_dataframe(pmc_corpus_count, "output/pmc_total_count.tsv")
+    pmc_corpus_count.most_common(10)
+
+
+# In[5]:
+
+
+if not nytac_count_path.exists():
+    nytac_corpus_count = (
+        aggregate_word_counts(
+            list(Path("../../nytac/corpora_stats/output").rglob("*tsv"))
+        )
+    )
+    dump_to_dataframe(nytac_corpus_count, "output/nytac_total_count.tsv")
+    nytac_corpus_count.most_common(10)
+
+
+# In[6]:
+
+
+biorxiv_total_count_df = pd.read_csv(
+    biorxiv_count_path.resolve(), 
+    sep="\t"
+)
+
+pmc_total_count_df = pd.read_csv(
+    pmc_count_path.resolve(), 
+    sep="\t"
+)
+
+nytac_total_count_df = pd.read_csv(
+    nytac_count_path.resolve(), 
+    sep="\t"
+)
+
+
+# In[7]:
+
+
+biorxiv_sentence_length = pickle.load(
+    open("output/biorxiv_sentence_length.pkl", "rb")
+)
+pmc_sentence_length = pickle.load(
+    open("../../pmc/pmc_corpus/pmc_sentence_length.pkl", "rb")
+)
+nytac_sentence_length = pickle.load(
+    open("../../nytac/corpora_stats/nytac_sentence_length.pkl", "rb")
+)
+
+
+# In[8]:
+
+
+spacy_nlp = spacy.load('en_core_web_sm')
+stop_word_list = list(spacy_nlp.Defaults.stop_words)
+
+
+# ## Get Corpora Comparison Stats
+
+# In[9]:
+
+
+biorxiv_sentence_len_list = list(biorxiv_sentence_length.items())
+biorxiv_data = {
+    "document_count": len(biorxiv_sentence_length),
+    "sentence_count": sum(map(lambda x: len(x[1]), biorxiv_sentence_len_list)),
+    "token_count": biorxiv_total_count_df['count'].sum(),
```

Comment:
>Is it worth printing out an example biorxiv_total_count_df since you're using it to get various word distributions here?

A few cells down I actually print out the full table.

Reply:
Sorry this question was not super clear. I guess what I was more asking was to see what the df looked like before your malnipulations, as I was interested in trying to learn more about how these pandas functions are working. But that was more for me..these are alot of chained pandas functions for me to digest:) If you have a screenshot of it for me to look at that might be helpful, but don't think you need to permanently change your code to print these df  out

Reply:
Ah I gotcha. Will redirect this one offline as this is more centered around the package rather than code within this PR.